src/multiplayer_server.cpp

```diff
@@ -513,8 +500,9 @@ void GameServer::handleNewClient(ClientInfo& info)
     //  For each component type, send all existing components.
     for(auto& ecsrb : sp::ecs::MultiplayerReplication::list)
         ecsrb->sendAll(ecs_packet);
+
     sendDataCounter += ecs_packet.getDataSize();
-    info.socket->queue(ecs_packet);
+    send_packet(ecs_packet);
```

Comment:
This isn't correct, send_packet sends to all clients, and this packet is only intended for this specific client.